### PR TITLE
Add main wrapper

### PR DESCRIPTION
**Pull request by frost-nzcr4**
Use `main()` so import of your package as third-party library doesn't hurt anyone. See https://stackoverflow.com/questions/419163/what-does-if-name-main-do

### CODE REVIEW

File: SMOKEnGAS.py [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
+            try:
+                with open('pms5003.csv', 'a', newline='') as csvfile:
+                    writer = csv.writer(csvfile, delimiter=' ', quotechar='|', quoting=csv.QUOTE_MINIMAL)
+                    readings = pms5003.read()  # PMS5003Data instance
+                    dt = datetime.now()
+                    readings1 = pms5003.read()
+                    readings2 = gas.read_all() #gas data instance
+                    writer.writerow([dt.isoformat()] + list(readings1.data[:-2] + [readings2.oxidising, readings2.reducing, readings2.nh3, readings2.adc])
+            except ReadTimeoutError:
+                pms5003 = PMS5003()
```
Comment (frost-nzcr4) [inline SMOKEnGAS.py:32]: You may require another `time.sleep(1.0)` here, to allow the device to initialize.

**Comment by swiss-JF** [comment]
> Traceback (most recent call last):
>   File "/home/pi/Documents/SMOKEnGAS/SMOKEnGAS.py", line 37, in <module>
>     main()
>   File "/home/pi/Documents/SMOKEnGAS/SMOKEnGAS.py", line 29, in main
>     writer.writerow([dt.isoformat()] + list(readings1.data[:-2] + [readings2.oxidising, readings2.reducing, readings2.nh3, readings2.adc]))
> TypeError: can only concatenate tuple (not "list") to tuple

I reinstall the Github enviroplus library. If I run 

> python gas.py

I can see the value, but when I run the SMOKEnGAS.py I get the error with tuple and list. 

I cannot look at it further, because I have to go to work, but somewhat getting there...

**Comment by frost-nzcr4** [comment]
Did you inistall it with `pip install enviroplus`? If so, you have to use `from enviroplus import gas`.

**Comment by swiss-JF** [comment]
I had installed from their github. If I type:
'''pip install enviroplus''', it gives me
'''Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Requirement already satisfied: enviroplus in /usr/local/lib/python2.7/dist-packages/enviroplus-0.0.3-py2.7.egg (0.0.3)
Requirement already satisfied: ads1015 in /usr/local/lib/python2.7/dist-packages/ads1015-0.0.5-py2.7.egg (from enviroplus) (0.0.5)
Requirement already satisfied: astral in /usr/local/lib/python2.7/dist-packages/astral-2.1-py2.7.egg (from enviroplus) (2.1)
astral requires Python '>=3.6,<4.0' but the running Python is 2.7.16''''

So, I think it is not the issue, which may be more at line 30.
If I replace list by tuple, I get
'''Typeerror: can only concatenate tuple (not "list") to tuple...

and I got it! I put a list for the small particles value and a list for the gas value, and it is up and running. :)

**Comment by swiss-JF** [comment]
Thank you for your help!

**Pull request merged**